Processes/DevRel-Publication-Process.md

```diff
@@ -0,0 +1,169 @@
+# Process: Publish New or Updated Content to the Dev Portal
+
+## Overview
+This document provides instructions on how to publish new or updated content to the EOS Dev Portal (docs.eosnetwork.com).
+
+**Note**: New or updated content is published to a private staging environment every day, which allows for content review prior to publication of content to the public dev portal. Once content is reviewed and approved for public consumption, the content can be published to the public dev portal. This publication process can be scheduled or ad hoc, depending on the need for the community to consume the content.
+
+### When to Use this Procedure
+Use this procedure when you need to update content from multiple repositories to the [EOS Dev Portal](http://docs.eosnetwork.com).
+
+### Expected Results
+Fresh, updated content is available for consumption on the EOS Dev Portal.
+
+# Procedures Contained in This Publication Process
+Complete the following procedures to publish new or updated content to the EOS Dev Portal:
```

Comment:
These steps don't flow properly and is dependent on whether the contributor is internal or external

External contributor (not part of this org)
1. Fork the docs repository
2. Create a new branch
2. Create/edit/rename files
4. Create a Pull Request with the target as this repository's main branch
5. Request a review

Internal contributor (part of this org)
1. Create a new branch
2. Create/edit/rename files
3. Create a Pull Request with main branch
4. Request a review

Reply:
Done